dev/cf_cli/cf_cli.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit -o nounset
+
+namespace="scf"
+deployment_name="scf"
+pod_name="cf-terminal"
+api_ip=""
+echo "Waiting for endpoint..."
+while true; do
```

Comment:
WIBNI (extension wish): auto-check for active ingress and use it if present. Alternatively, script option.

Reply:
The purpose of this script isn't to talk to ingress, but to allow us to test other aspects of the CF cluster.